### ISSUE TITLE
Maven... really?

### ISSUE THREAD

**Issue by Redstone-Pro-73** [OgarProject/Ogar2-Server#29]
Can you please just have a jar file for people to download. I can not describe the time I have wasted and the frustration I've been through. Maven never seems to work for me, and installing it seems like to most complicated thing in the world. Please make it easier for everyone with just a jar file download. If for some reason you can not supply me with the jar, please give me instructions (step by step) on how to install maven on my Windows 10 PC or Cent OS 6 server and then compile the Ogar project. Thanks

**Comment by ThunderGemios10** [comment]
@Redstone-Pro-73 Check our version of Ogar2, It has more features than this.. https://github.com/OgarioProject/Ogar2-Server/ Also we have a pre-compiled jar. https://ci.thundergemios10.com/job/Ogar2-Server

**Comment by Redstone-Pro-73** [comment]
@ThunderGemios10 Thank you so much! Just what I needed

**Comment by Redstone-Pro-73** [comment]
@ThunderGemios10 Actually, it seems your website is down. I've refreshed it periodically for around an hour, and still nothing. Just the cloudflare error 522 comes up. Hopefully you can fix it so tomorrow I can download the compiled version. Thx

**Comment by ThunderGemios10** [comment]
@Redstone-Pro-73, Seems my server overloaded xD, Resolving it right now.
EDIT: So it turns out, DigitalOcean suspended my account for not sending a payment.. So unfortunately you may have to compile it yourself. But don't worry, I'll put some compiled jars at "releases" section,.

**Comment by ThunderGemios10** [comment]
@Redstone-Pro-73 Nevermind, I contacted them and they enabled back my account and droplet. Website up so you can download the compiled jar.

**Comment by Redstone-Pro-73** [comment]
@ThunderGemios10 Seems it went down again. I wasn't able to download in the morning, so I was wondering if you could set the server back up. Or if it is easier, just send me the file through something like mediafire. Thanks for your help

**Comment by ThunderGemios10** [comment]
@Redstone-Pro-73 I snapshot my droplet and took it down. Will restore once i pay up. Don't worry i'll put up a download for the jars.

**Comment by ThunderGemios10** [comment]
@Redstone-Pro-73 Ok, Download is available, Check https://github.com/OgarioProject/Ogar2-Server/releases

**Comment by Redstone-Pro-73** [comment]
Thank you! Just what I needed! Great support! 8/8 Absolutely GR8 M8!

**Comment by Redstone-Pro-73** [comment]
@ThunderGemios10 I don't need this right away, but could you also please compile the API. I would like to start making plugins and it looks similar to making MC plugins, which I already have a lot of experience with.

**Comment by ThunderGemios10** [comment]
@Redstone-Pro-73 Yup! Ogar2 is heavily inspired by Bukkit, Also would you like help out developing Ogar2? Our team are slacking most of the time and we need all the help, Also sure! I'll compile the API.
